unified_targets/configs/MAMBAF722.config

```diff
@@ -0,0 +1,114 @@
+# Betaflight / STM32F7X2 (S7X2) 4.0.1 May  1 2019 / 22:19:19 (25f4a5ffe) MSP API: 1.41
+
+board_name MAMBAF722
+manufacturer_id DIAT
+
+# I/O
+resource LED 1 C15
+resource LED 2 C14
+resource BEEPER 1 B02
+set beeper_inversion = ON
+set beeper_od = OFF
+
+# ACC/GYRO
+resource GYRO_EXTI 1 C04
+resource GYRO_CS 1 A04
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
+set gyro_1_sensor_align = CW180
+
+# SPI
+resource SPI_SCK 1 A05
+resource SPI_MISO 1 A06
+resource SPI_MOSI 1 A07
+
+resource SPI_SCK 2 B13
+resource SPI_MISO 2 B14
+resource SPI_MOSI 2 B15
+
+resource SPI_SCK 3 C10
+resource SPI_MISO 3 C11
+resource SPI_MOSI 3 B05
+
+# FLASH
+resource FLASH_CS 1 A15
+set flash_spi_bus = 3
+
+# OSD
+resource OSD_CS 1 B12
+set max7456_spi_bus = 2
+
+# ADC
+resource ADC_BATT 1 C01
+resource ADC_RSSI 1 C02
+resource ADC_CURR 1 C03
+set adc_device = 3
+
+# SERIAL PORTS
+resource SERIAL_TX 1 B06
+resource SERIAL_RX 1 B07
+
+resource SERIAL_TX 2 A02
+resource SERIAL_RX 2 A03
+
+resource SERIAL_TX 3 B10
+resource SERIAL_RX 3 B11
+
+resource SERIAL_TX 4 A00
+resource SERIAL_RX 4 A01
+
+resource SERIAL_TX 5 C12
+resource SERIAL_RX 5 D02
+
+resource SERIAL_TX 6 C06
+resource SERIAL_RX 6 C07
+
+# TIMERS
+timer B09 1
+timer C08 1
+timer C09 1
+timer A08 0
+timer A09 0
+timer B08 0
+timer B03 0
+resource MOTOR 1 C08
+resource MOTOR 2 C09
+resource MOTOR 3 A08
+resource MOTOR 4 A09
+resource PPM 1 B09
+resource LED_STRIP 1 B03
+resource CAMERA_CONTROL 1 B08
+
+# dma
+dma ADC 3 0    # ADC 3: DMA2 Stream 0 Channel 2
+dma pin C08 0  # pin C08: DMA2 Stream 2 Channel 0
+dma pin C09 0  # pin C09: DMA2 Stream 7 Channel 7
+dma pin A08 0  # pin A08: DMA2 Stream 6 Channel 0
+dma pin A09 0  # pin A09: DMA2 Stream 6 Channel 0
+dma pin B08 0  # pin B08: DMA1 Stream 7 Channel 2
+dma pin B03 0  # pin B03: DMA1 Stream 6 Channel 3
+
+# serial
+serial 0 64 115200 57600 0 115200
+serial 3 1 19200 57600 0 115200
+
+# EXTRAS
+resource ESCSERIAL 1 B09
+resource PINIO 1 B00
+
+# feature
+feature RX_SERIAL
+feature TELEMETRY
+feature SOFTSERIAL
+feature OSD
+
+# master
+set mag_hardware = NONE
+set baro_hardware = NONE
+set serialrx_provider = SBUS
+set blackbox_device = SPIFLASH
+set dshot_burst = ON
+set motor_pwm_protocol = DSHOT600
+set current_meter = ADC
+set battery_meter = ADC
+set system_hse_mhz = 8
```

Comment:
This is definitely not needed for F7.

Reply:
I’ll take out the DSHOT as requested.   Diatone will be using their boards mostly in complete drone with DSHOT600 the default so decided to put that in for ease of the consumer.  

Question about the others though.   These are spit out from the CLI using the instructions given and are settings in the configurator.  I’ve checked the other F7 config files and they all seem to have them as well.

Reply:
Sorry, I was only talking about `system_hse_mhz`, which is not even used on F7. And you are right, all (or most) Unified Target configs have this at the moment - I'll have to investigate why this is printed in the `diff`, since 8 MHz is the default.

Reply:
@mikeller No worries.   My bad for not understanding what you were referring to.  Appreciate the response.   When I get home from work, I'll add back in the DSHOT per the request of the manufacturer and repush if that works for you.  Let me know if I need to clean anything else up.  Much appreciated.  Great work on the Unified Target concept!

Reply:
Target change made as required.  Thanks.